usaspending_api/awards/v2/urls_awards.py

```diff
@@ -13,10 +13,21 @@
     url(r"^funding/$", AwardFundingViewSet.as_view()),
     url(r"^funding_rollup/$", AwardFundingRollupViewSet.as_view()),
     url(r"^last_updated", AwardLastUpdatedViewSet.as_view()),
-    url(r"^(?P<requested_award>[A-Za-z0-9_. -]+)/$", AwardRetrieveViewSet.as_view()),
-    url(r"^count/transaction/(?P<requested_award>[A-Za-z0-9_. -]+)/$", TransactionCountRetrieveViewSet.as_view()),
-    url(r"^count/subaward/(?P<requested_award>[A-Za-z0-9_. -]+)/$", SubawardCountRetrieveViewSet.as_view()),
+    url(r"^count/transaction/(?P<requested_award>[0-9]+)/$", TransactionCountRetrieveViewSet.as_view()),
     url(
-        r"^count/federal_account/(?P<requested_award>[A-Za-z0-9_. -]+)/$", FederalAccountCountRetrieveViewSet.as_view()
+        "^count/transaction/(?P<requested_award>(CONT|ASST)_(AWD|IDV|NON|AGG)_.+)/$",
```

Comment:
Why `.+` instead of `.*`?

Reply:
.* is 0 or more, .+ is one or more. we don't have or want any award that's just `CONT_AWD_`.